src/domain/Tables/Table/Table.tsx

```diff
@@ -180,66 +142,62 @@ const Table = <T extends{}>(props: ITableProps<T>) => {
             sort={sort}
             onSortChange={onSortChange}
             columns={columns}
-            selectedAll={selectedAll}
-            setSelectedAll={setSelectedAll}
+            selectAllTableCheckboxInputValue={selectAllTableCheckboxInputValue}
+            setSelectAllTableCheckboxInputValue={setAllRows}
             hasLeftAction={hasLeftAction}
             bulkActions={bulkActions}
             hasTableSwipeActions={hasTableSwipeActions}
-            hasBulkAction={selectedAll !== TableCheckboxInputValue.False}
+            hasBulkAction={selectAllTableCheckboxInputValue !== TableCheckboxInputValue.False}
             isEmpty={rows.length === 0}
             hasSortEnabled={hasSortEnabled}
           />
         </StyledTHead>
-        <TransitionGroup className='table-rows' component='tbody'>
           {
             rows.length === 0 ? (
-              <StyledEmptyStateRow><StyledEmptyStateCell colSpan={columns.length}>{emptyState}</StyledEmptyStateCell></StyledEmptyStateRow>
-            ) : rows.map((row: IRowProps<T>, index) => (
-                <Transition
-                  key={row.id}
-                  timeout={500}
-                  mountOnEnter
-                  unmountOnExit
-                >
```

Comment:
React-level transitions were removed, because they cause issues when going between multiple pages as it'll animate in and out the rows incorrectly.